### PR TITLE
Improved: Cleaned commented code from entity component (OFBIZ-8038)

### PR DESCRIPTION
**Pull request by ieugen**
Improved: Cleaned commented code from entity component (OFBIZ-8038)

### CODE REVIEW

**Comment by ieugen** [comment]
Some comments might be reverted. 
They seem to be used for debugging.
I removed them to make them stand out for review.

File: framework/entity/src/main/java/org/apache/ofbiz/entity/model/ModelReader.java [hunk 1]
```diff
@@ -225,8 +224,7 @@ private ModelEntity buildEntity(ResourceHandler entityResourceHandler, Element c
                             throw new GenericEntityConfException("Could not get document for " + entityResourceHandler.toString());
                         }
 
-                        // utilTimer.timerString("Before getDocumentElement in " +
```
Comment (JacquesLeRoux) [inline framework/entity/src/main/java/org/apache/ofbiz/entity/model/ModelReader.java:228]: Hi Eugen,

Do we really need this one?

File: framework/entity/src/main/java/org/apache/ofbiz/entity/util/EntityCrypto.java [hunk 2]
```diff
@@ -110,16 +110,6 @@ public String encrypt(String keyName, EncryptMethod encryptMethod, Object obj) t
         }
     }
```
Comment (JacquesLeRoux) [inline framework/entity/src/main/java/org/apache/ofbiz/entity/util/EntityCrypto.java]: I think we need to keep this one

Comment (ieugen) [reply framework/entity/src/main/java/org/apache/ofbiz/entity/util/EntityCrypto.java]: Added back.

File: framework/entity/src/main/java/org/apache/ofbiz/entity/model/ModelReader.java (hunk 1 above)
Comment (ieugen) [reply framework/entity/src/main/java/org/apache/ofbiz/entity/model/ModelReader.java:228]: I've kept one line comments that seem to be useful in debugging. 
They are not that many so have little impact.

**Comment by ieugen** [comment]
Ready for merge.

**Pull request merged**